### ISSUE TITLE
Request: Have <link rel="schema.dcterms" href="http://purl.org/dc/terms/" /> added to WET

### ISSUE THREAD

**Issue by SimonaIoffe** [wet-boew/wet-boew#7872]
Currently schema.DCTERMS works but schema.dc/schema.DC doesn’t is because only schema.DCTERMS seems to be registered as a valid HTML link type extension (http://microformats.org/wiki/existing-rel-values#HTML5_link_type_extensions). 

This might be because "Elements from /elements/1.1/ namespace have not been included on purpose. According to "FAQ/DC and DCTERMS Namespaces", the old namespace is maintained for legacy purposes only and it is not as suitable for non-literal values as the ones denoted by <link> elements."

I would like to request that <link rel="schema.dcterms" href="http://purl.org/dc/terms/" /> be added to WET.

Many thanks in advance!

**Comment by nschonni** [comment]
Think you'd need to get the [Standard on Web Interoperability](http://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=25875) changed for that. It would likely be something that would end up in GCWeb rather than the WET core

**Comment by LaurentGoderre** [comment]
Agreed! WET should stay lightweight and decoupled from GC Standards

**Comment by SimonaIoffe** [comment]
Thanks Nick/Laurent. I don't disagree. This is part of a bigger discussion.
